hydra-node/json-schemas/api.yaml

```diff
@@ -806,7 +806,22 @@ components:
                   },
                   "redeemer": "21"
               }
-            }
+            },
+            "0700070603080308050807020202020806080000030605030704010503040505#10": {
+                "address": "addr_test1xrfd3ey8p85wcx8rhk8hxtrdgw580ukfsa9f2kqs7hpw5hkmyp5cm0cvpt6hx5vd34fvjtk07uxlkw6f8g0ewamhwuds9p7e97",
+                "datum": null,
+                "inlineDatum": {
+                    "bytes": "8efa4f"
+                },
+                "inlineDatumhash": "634530d5bd45fb5ef71043bfa41a4144b66e2f0a4e0a1f7468ead6c22be5584a",
+                "referenceScript": null,
+                "value": {
+                    "467f58932b54910584a0e8ea25a225e06a14530b2e96e938c53a3f22": {
+                        "3fba8d7964f160a41a26504b2f": 4376550814605889237
+                    },
+                    "lovelace": 5787458043450694813
+                }
```

Comment:
This example would not work. It's missing a `witness` to spend the script output. We still need a witness with `plutusScript` and `redeemer` inside.

Reply:
I added the examples using _real_ utxo